bucket/hydrus-network.json

```
@@ -0,0 +1,44 @@
+{
+    "version": "377",
+    "description": "A personal booru-style media tagger",
+    "homepage": "https://hydrusnetwork.github.io/hydrus/",
+    "license": "WTFPL",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/hydrusnetwork/hydrus/releases/download/v377/Hydrus.Network.377.-.Windows.-.Extract.only.zip",
+            "hash": "cf8a0820d4e98e4ea6b07ae51714510ce76bfdab40ca104fb0b2db9c4ca9a6b7"
+        }
+    },
+    "extract_dir": "Hydrus Network",
```

Add something like following to preserve updates to sqllite executable:


```suggestion
    "extract_dir": "Hydrus Network",
    "post_install": "if (Test-Path \"$dir\\db.original\\sqlite3.exe\") { Copy-Item \"$dir\\db.original\\sqlite3.exe\" \"$dir\\db\" -Force }",

```